css/style.css

```diff
@@ -43,6 +42,121 @@ body {
   margin: auto;
   height: 500px;
 }
+.logoOne {
+  margin-left: 100px;
+  font-size: 24px;
+  font-weight: 900;
+  color: white;
+}
+.message {
+  text-align: center;
+  font-weight: bold;
+  font-size: 24px;
+  color: white;
+  margin-top: 150px;
+}
+.centerButton { 
+  text-align: center;
+  margin-top: 50px;
+}
+.getIt {
```

Comment:
Great use of camel case class names!

https://en.wikipedia.org/wiki/Camel_case